packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -67,6 +76,32 @@ describe('App', () => {
         expect(screen.getByText('Any children here')).toBeInTheDocument()
     })
 
+    test('Active Data component is not rendered', () => {
+        mockActiveDataEnable = false
+        useMultiSite.mockImplementation(() => resultUseMultiSite)
+        renderWithProviders(
+            <App targetLocale={DEFAULT_LOCALE} defaultLocale={DEFAULT_LOCALE} messages={messages}>
+                <p>Any children here</p>
+            </App>
+        )
+        waitFor(() => expect(document.getElementById('#headActiveData')).not.toBeInTheDocument())
+        waitFor(() => expect(document.getElementById('#dwanalytics')).not.toBeInTheDocument())
+        waitFor(() => expect(document.getElementById('#dwac')).not.toBeInTheDocument())
```

Comment:
```suggestion
        await waitFor(() => expect(document.getElementById('#headActiveData')).not.toBeInTheDocument())
        await waitFor(() => expect(document.getElementById('#dwanalytics')).not.toBeInTheDocument())
        await waitFor(() => expect(document.getElementById('#dwac')).not.toBeInTheDocument())
```
`waitFor` is async, so it _must_ be `await`ed, otherwise the test will always be a false positive.

Reply:
done